clients/aws-sdk-transcribe-streaming/src/aws_sdk_transcribe_streaming/config.py

```diff
@@ -0,0 +1,200 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+from dataclasses import dataclass
+from typing import Any, Callable, TypeAlias, Union
+
+from smithy_aws_core.aio.protocols import RestJsonClientProtocol
+from smithy_aws_core.auth import SigV4AuthScheme
+from smithy_aws_core.endpoints.standard_regional import (
+    StandardRegionalEndpointsResolver as _RegionalResolver,
+)
+from smithy_aws_core.identity import AWSCredentialsIdentity, AWSIdentityProperties
+from smithy_core.aio.interfaces import (
+    ClientProtocol,
+    ClientTransport,
+    EndpointResolver as _EndpointResolver,
+)
+from smithy_core.aio.interfaces.auth import AuthScheme
+from smithy_core.aio.interfaces.identity import IdentityResolver
+from smithy_core.interceptors import Interceptor
+from smithy_core.interfaces import URI
+from smithy_core.interfaces.retries import RetryStrategy
+from smithy_core.retries import SimpleRetryStrategy
+from smithy_core.shapes import ShapeID
+from smithy_http.aio.crt import AWSCRTHTTPClient
+from smithy_http.interfaces import HTTPRequestConfiguration
+
+from ._private.schemas import TRANSCRIBE as _SCHEMA_TRANSCRIBE
+from .auth import HTTPAuthSchemeResolver
+from .models import (
+    GetMedicalScribeStreamInput,
+    GetMedicalScribeStreamOutput,
+    StartCallAnalyticsStreamTranscriptionInput,
+    StartCallAnalyticsStreamTranscriptionOutput,
+    StartMedicalScribeStreamInput,
+    StartMedicalScribeStreamOutput,
+    StartMedicalStreamTranscriptionInput,
+    StartMedicalStreamTranscriptionOutput,
+    StartStreamTranscriptionInput,
+    StartStreamTranscriptionOutput,
+)
+
+
+_ServiceInterceptor = Union[
+    Interceptor[GetMedicalScribeStreamInput, GetMedicalScribeStreamOutput, Any, Any],
+    Interceptor[
+        StartCallAnalyticsStreamTranscriptionInput,
+        StartCallAnalyticsStreamTranscriptionOutput,
+        Any,
+        Any,
+    ],
+    Interceptor[
+        StartMedicalScribeStreamInput, StartMedicalScribeStreamOutput, Any, Any
+    ],
+    Interceptor[
+        StartMedicalStreamTranscriptionInput,
+        StartMedicalStreamTranscriptionOutput,
+        Any,
+        Any,
+    ],
+    Interceptor[
+        StartStreamTranscriptionInput, StartStreamTranscriptionOutput, Any, Any
+    ],
+]
+
+
+@dataclass(init=False)
+class Config:
+    """Configuration for Transcribe."""
```

Comment:
What'd we use to generate this? It's strange we've named the client correctly (`TranscribeStreamingClient`) but this is `Transcribe` which would be a different client. It seems like we potentially have a mismatch or Transcribe themselves may not be making the distinction we expect. Is this the signing name?